### DISCUSSION TITLE
[Artillery /w Playwright] Firefox / Safari support question

### DISCUSSION THREAD

**Discussion by EmmanH** [artilleryio/artillery#1981] [Artillery]
Hello Artillery team, 
As Playwright user i do find the extension with artillery quite useful and powerful. 
My only problem is i can't see a way to do perf testing on safari / firefox. 
I am not sure if it's because it's not available or didn't search enough to get the information (I tried different things but none of them work so far).

Is it possible to do it on those browsers?

Thank you,

**Comment by hassy** [comment]
Hi @EmmanH 👋 currently only Chromium is supported. Do you expect load testing results to vary significantly between Chrome and Firefox/Safari?

**Comment by EmmanH** [comment]
Thank you for your quick answer! 
It could yes but more importantly for the performance metrics I do within my company, i can't give the chromium perf for the firefox/safari one hence my question :) 
In that case, is there any alternative to measure it? :)
